scripts/packaging/createupmpackages.ps1

```diff
@@ -159,11 +159,29 @@ foreach ($entry in $packages.GetEnumerator()) {
         $registryName = "the $OutputTarget registry"
     }
 
-    # The examples folder needs a custom setup step
-    if ($entry.Name -eq "examples") {
-        Write-Output "Preparing the examples folder"
+    $samplesFolder = "$packagePath\Samples~"
+     
+    if ($packageName -eq "examples") {
+        # The examples folder needs a custom setup step

```

Comment:
Nit: Can you expand on the comment to explain why it needs a custom step?

Reply:
sure

Reply:
updated